src/index.tsx

```diff
@@ -8,6 +8,11 @@ if (typeof(document) === 'undefined') {
   throw new Error('Browser entrypoint must be used in the browser');
 }
 
+if (window && window.top === window.self) {
```

Comment:
should do 

```
typeof(window) !== 'undefined'
```

not that it matters because this file will always be in the browser, but i'd like to get in the habit of doing the checks correctly

Reply:
I wonder, what is the advantage of checking the type of window vs just checking for the existence of it?

Reply:
`if (window)` will throw an error if window is not set, which is the case in node. `typeof(window) !== 'undefined'` will not throw an error if window is unset